### ISSUE TITLE
Feature request: CLI version of design-tokens

### ISSUE THREAD

**Issue by marianntapfer** [lukasoppermann/design-tokens#223]
Would you consider adding a CLI version to your plugin? 
It would make more sense on the developer side if it would be possible to provide a Figma access token, file id and run this locally to get the same kind of json file I get from the plugin in Figma. 

What would be the benefits of using a CLI version: 

1. We have been using your Figma plugin for a while now and it is great the only downside is that if something changes there is no way for us to control the version of the plugin - it updates automatically. 

2. For us it is usually the developer who exports the json converts it to tokens using style-dictionary and adds that change to the library, but the dev usually does not have edit right on the Figma file - so a workaround id to duplicate the file and then on my draft I can use the plugin. This means I need to double check the settings every time. I think it should be possible to get data trough Figma api from a file where I don't have edit rights.

**Comment by lukasoppermann** [comment]
Hey, 

I have not considered this. But it sounds like a pretty good idea. 

Did you build a prototype already?

**Comment by marianntapfer** [comment]
No prototype jet just an idea, but I would definitely be up to contributing.

**Comment by lukasoppermann** [comment]
That would be great. I think it is a very nice idea as it opens up integration into tool chains without user interaction.

Would you like to start a prototype? Do you need anything to get started?

**Comment by marianntapfer** [comment]
Yes. IM not sure when I will have to get to it, but I will keep you posted. Very nice to know you are open to the idea.
